rfc/text/advanced/ap_27_pubsub_event_history.md

```diff
@@ -1,58 +1,132 @@
 ## Event History {#pubsub-event-history}
 
-Instead of complex QoS for message delivery, a *Broker* may provide *message history*. A *Subscriber* is responsible to handle overlaps (duplicates) when it wants "exactly-once" message processing across restarts.
-
-The *Broker* may allow for configuration on a per-topic basis.
+Instead of complex QoS for message delivery, a *Broker* may provide *message history*. A *Subscriber* is 
```

Comment:
yes, that is actually one argument when looking at "event history" as an alternative to what QoS'ed PubSub provides. in my mind, the word "Instead" however suggests it's either/or .. while IMO there are arguments for both. As in: even when an implementation provides event history, QoS for PubSub might still make sense. And WAMP does have acknowledged PubSub (QoS-1) at least. And there is a proposal / exp. impl for `EVENT_RECEIVED`  https://github.com/crossbario/autobahn-python/blob/f77396401d52a5284c3506171cf45eb64505804f/autobahn/wamp/message.py#L3873 which would allow higher QoS levels. 

So IMO it would be great to add a bit more text for motivation, context/background and related features or feature proposals. I think it is crucial for a reader to have a good understanding of our original motivation and thinking.

Reply:
👍🏻  Agree. I personally don't compare event history with QoS at all. MB I'll provide some other use cases.